packages/installation-tests/sanity.js

```diff
@@ -19,6 +19,10 @@ const browsers = process.argv.slice(3);
 
 const playwright = require(requireName);
 
+// Requiring internals should work.
+const errors = require(requireName + '/lib/errors');
+const installer = require(requireName + '/lib/install/installer');
```

Comment:
```
const rpcServer = require(requireName + '/lib/rpc/server');
```
lets add this too?

Reply:
I tried at first, but it actually starts the server, so I cannot do that hermetically in the test 😄